docker/image_test.go

```diff
@@ -37,3 +41,15 @@ func TestParseImageFromName(t *testing.T) {
 		}
 	}
 }
+
+func TestParseImageFromRepoAndSha(t *testing.T) {
+	registries := []string{"", "quay.io"}
```

Comment:
fixed in https://github.com/deis/deisrel/pull/109/commits/aa839f71014a0ca61d31c0714a2b172b68c37630